src/index.js

```diff
@@ -7,6 +7,23 @@ import { StoreDetails } from "./components/StoreDetails";
 import { WonderCart } from "./components/WonderCart";
 import { createApiUrl } from "./sdk/createApiUrl";
 import { PluginsSdk } from "./sdk/plugins";
+import { HiiveAnalytics } from '@newfold-labs/js-utility-ui-analytics';
+import domReady from '@wordpress/dom-ready';
```

Comment:
Can we add `@wordpress/dom-ready` to dev dependencies?

Reply:
done